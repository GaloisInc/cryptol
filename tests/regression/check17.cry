check17 = ("" : [0][8]) == []
