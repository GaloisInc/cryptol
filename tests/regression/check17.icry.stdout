Loading module Cryptol
Loading module Cryptol
Loading module [check17.cry]
True
